shared/js/ui/platform-features.mjs

```diff
@@ -40,6 +40,7 @@ export function createPlatformFeatures(platform) {
         supportsHover: desktop.includes(platform.name),
         initialScreen: screen,
         supportsInvalidCerts: platform.name !== 'browser' && platform.name !== 'windows',
+        supportsPhishingWarning: platform.name === 'macos', // TODO: Add a version number?
```

Comment:
This 'todo' comment, is it because Thom's work is not being released to all macOS versions?

Reply:
No, I just wanted to check whether we need to lock it to a minimum version. But now that I think about it, previous versions wouldn’t carry this version of the C-S-S, so my question doesn’t really make sense.

Reply:
Comment removed!